src/styles/01-colors/_colors-utility.css

```diff
@@ -13,9 +20,17 @@
   --ui-gray-xxlight: #fafafa;
 
   /** State and link utilities */
-  --ui-success: #077719;
-  --ui-error: #97272c;
   --ui-focus: #4181f1;
   --ui-link-primary: #0576d3;
   --ui-link-secondary: #004b98;
+  --ui-success-primary: #077719;
+  --ui-success-secondary: #095212;
+  --ui-warning-primary: #f0974e;
+  --ui-warning-secondary: #ec7b1f;
+  --ui-status-primary: #f9e08e;
+  --ui-status-secondary: #fbe7e1;
+  --ui-error: #97272c;
+  --ui-disabled-primary: #e0e0e0;
+  --ui-disabled-secondary: #fafafa;
+  --ui-test: #ff6347;
 }
```

Comment:
Consider alphabetizing the variables within each category  --  we actually have a linter setting that recommends doing that.  Let me know if you'd like to set that up in your IDE if you haven't already.

Reply:
@kaplanmichael 
OK, I alphebetized the vars where it made sense.  I did not touch the ordering of the grayscale vars, as they are curerently ordered based on the color values from dark to light.